semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/SymbolOps.scala

```diff
@@ -23,15 +23,20 @@ trait SymbolOps { self: DatabaseOps =>
           def definitelyLocal =
             sym == g.NoSymbol ||
               sym.name.decoded.startsWith(g.nme.LOCALDUMMY_PREFIX) ||
-              (sym.owner.isMethod && !sym.isParameter) ||
-              ((sym.owner.isAliasType || sym.owner.isAbstractType) && !sym.isParameter)
+              sym.owner.isMethod ||
+              sym.owner.isAliasType ||
+              sym.owner.isAbstractType
           !definitelyGlobal && (definitelyLocal || isLocal(sym.owner))
         }
         if (isLocal(sym)) {
           val mpos = sym.pos.toMeta
           return {
             if (mpos == m.Position.None) m.Symbol.None
-            else m.Symbol.Local(mpos.syntax)
+            else {
+              val id = idCache.get(mpos.input.syntax)
+              idCache.put(mpos.input.syntax, id + 1)
+              m.Symbol.Local("local" + id.toString)
```

Comment:
I wouldn't be surprised if this change has a positive impact on file size ^^

Reply:
Later on, we could make it even shorter, e.g. `%0`, `%1` etc like in LLVM.